### DISCUSSION TITLE
Volume Amplification (Over 100% Volume causing static noise)

### DISCUSSION THREAD

**Discussion by vetux** [falkTX/Carla#1540] [Q&A] [answered]
I am surprised noone asked or posted an issue about this yet when using carla with pulseaudio backend the instruments can output volume which is over 100% (Visible as the yellow / red coloring of the volume bars) How can i prevent this from happening? I want carla to only output 100% maximum volume not more because it causes static noise.

**Comment by falkTX** [comment, marked as answer]
This is a common "issue" for everything that deals with audio.
Some DAWs/tools have hard-clipping in place, Carla (and a few others) do not.
So it will be your responsibility to ensure audio does not go above -1/+1 values (which matches 0dB)

Typically you just use a limiter plugin for this.

**Reply by vetux** [reply]
I see thanks for the reply, where could i find such a limiter plugin? I can pass the instrument output through a "Audio Gain" and slightly decrease volume but surely there is a better way? Sorry if i sound like a total idiot i am new to audio stuff.

**Reply by falkTX** [reply]
There is many of them, so it depends on what you have installed.
https://x42-plugins.com/x42/x42-limiter is a great one, usually part of the x42-plugins package.

**Reply by vetux** [reply]
Thanks a lot it works perfectly :)

**Reply by vetux** [reply]
After using the plugin x42 limiter plugin for some time i notice still very noticeable static when overamplifying but less than without the plugin is there a way to get rid of the static for good? And why is such a "limiter plugin" not part of the default plugins if the sound output is that bad without it which makes it useless without the plugin? In lmms for example i do not have any such issues with the vst instrument however latency and dsp spikes are unbearable in lmms.

**Reply by falkTX** [reply]
People have different tastes on what a limiter should do and behave, it is not up to me to force a dedicated one at the end of the chain.
Plus, most soundcards do not misbehave when receiving values that go above 0dB, they tend to have some tolerances. 
Your issue is actually special and unusual from what I have experience with other systems.

**Reply by vetux** [reply]
I see well i am using linux ([Carla 2.2.0](https://github.com/falkTX/Carla/releases/download/v2.2.0/Carla_2.2.0-linux64.tar.xz)) with the pulseaudio backend. The static actually seems to happen before it reaches pulseaudio/soundcard though if i dont use the limiter and turn down the volume either of the rack "item?" or the carla program volume in pavucontrol i still get static without the db bars in pavucontrol reaching 100%. However looking at carla volume bars the static happens exactly when the bars go into the yellow / red area when the instrument volume is at 100% in carla, regardless of the volume setting of the carla program or global volume in pavucontrol. With the limiter plugin the output volume bars of the limiter do not reach the yellow / red area however i still sometimes get static when the instrument volume bars go into the yellow / red area. Perhaps i should try windows?
